test/Test/Data/Mutable/Array.hs

```diff
@@ -64,6 +64,8 @@ group =
   , testProperty "∀ a,i,x. len (write a i x) = len a" lenWrite
   , testProperty "∀ a,s. len (resize s a) = s" lenResize
   , testProperty "∀ a,s,x. len (resizeSeed s x a) = s" lenResizeSeed
+  -- unit tests
```

Comment:
Nitpick: all the tests in this file are unit tests. I never remember a name for non-property based tests, though.

Reply:
Yeah, I agree. What if I just say "others"?

Reply:
Fair. Though, maybe, the point is that these are regression tests? So maybe “regression tests”?